src/vignette/api/legacy/routes.clj

```diff
@@ -172,6 +174,10 @@
   "Add the :width field to a request map based on the legacy parsing methods."
   (if-let [thumb-dimension (:dimension map)]
     (cond-let
+      ; this matches some legacy video requests that attempt to take the width from a frame in the "middle"
```

Comment:
The magic can be found around [here](https://github.com/Wikia/backend/blob/master/lib/Wikia/Thumbnailer/Video.pm#L28).